app/models.py

```diff
@@ -22,8 +16,61 @@ class User(Base):
     password = Column(String, nullable=False)
     first_name = Column(String, nullable=False)
     last_name = Column(String, nullable=False)
-    gender = Column(Enum("male", "female",), nullable=False)
+    gender = Column(Enum("male", "female", name="gender_enum"), nullable=False)
     age = Column(Integer, nullable=False)
     height = Column(Integer, nullable=False)
     weight = Column(Float, nullable=False)
+    activity_level = Column(Integer, nullable=False)
     user_created_at = Column(TIMESTAMP(timezone=True), nullable=False, server_default=text("NOW()"))
+
+class Tdee(Base):
+    __tablename__ = "tdees"
+    user_id = Column(Integer,ForeignKey("users.user_id", ondelete="CASCADE"), primary_key=True, nullable=False)
+    tdee_number = Column(Integer, nullable=False)
+    tdee_created_at = Column(TIMESTAMP(timezone=True), nullable=False,  primary_key=True, server_default=text("NOW()"))
+
+class Bmi(Base):
```

Comment:
What is the purpose of this table? Why you want to store BMI separately? Why not to create user_history table and store there all possible changes - for example, user can change activity level